dotcom-rendering/src/web/components/Discussion.tsx

```diff
@@ -180,62 +164,29 @@ export const Discussion = ({
 								/>
 							</div>
 						</Hide>
-
-						{beingHydrated && isExpanded && (
-							<Comments
-								user={user}
-								baseUrl={discussionApiUrl}
-								pillar={format.theme}
-								initialPage={commentPage}
-								pageSizeOverride={commentPageSize}
-								isClosedForComments={
-									isClosedForComments ||
-									!enableDiscussionSwitch
-								}
-								orderByOverride={commentOrderBy}
-								shortUrl={shortUrlId}
-								additionalHeaders={{
-									'D2-X-UID': discussionD2Uid,
-									'GU-Client': discussionApiClientHeader,
-								}}
-								expanded={isExpanded}
-								commentToScrollTo={hashCommentId}
-								onPermalinkClick={handlePermalink}
-								apiKey="dotcom-rendering"
-								onExpanded={(value) => {
-									handleExpanded(value);
-								}}
-							/>
-						)}
-
-						{beingHydrated && !isExpanded && (
-							<Lazy margin={300} disableFlexStyles={true}>
```

Comment:
We no longer need this Lazy component because Island provides this deferred loading built in